sql_generators/glean_usage/common.py

```diff
@@ -185,6 +185,7 @@ def __init__(self):
         self.across_apps_enabled = True
         self.cross_channel_template = "cross_channel.view.sql"
         self.base_table_name = "baseline_v1"
+        self.parallelism = 8
```

Comment:
`bqetl generate glean_usage` has a `--parallelism` parameter, so IMO it'd be better to use that rather than hard-coding the parallelism.